### ISSUE TITLE
第五周任务的小问题

### ISSUE THREAD

**Issue by Avalonere** [X-lab2017/dase-2023-autumn#21]
教材第136页第四题给出的程序似乎是选择排序而不是题目说的希尔排序
<img width="429" alt="image" src="https://github.com/X-lab2017/dase-2023-autumn/assets/144768885/f7265664-8240-47a7-8433-c74ca185f18c">

**Comment by andyhuang18** [comment]
是的...另外一个课程有同学反映，参见这个[issue](https://github.com/X-lab2017/ds-2023-autumn/issues/7#issuecomment-1736610539)。感谢你的反馈！

**Comment by k1nsom** [comment]
<img width="599" alt="image" src="https://github.com/X-lab2017/dase-2023-autumn/assets/144779386/09701149-c82c-46d2-bdf3-81c84c8a9390">

**Comment by lziyii** [comment]
第四章第四题给出的选择排序代码中，最后的return行应该少了一个缩进吧
